app/event/send-session-event.js

```diff
@@ -14,8 +14,6 @@ const sendSessionEvent = (organisation, sessionId, entryKey, key, value, ip, ref
       data: { reference, [key]: value },
       ip
     }
-    retryRaiseEvent(event, 2)
+    retryRaiseEvent(event, 2).then(r => {})
```

Comment:
what is this `.then(r => {})` doing for us?

Reply:
Ha I meant to remove that actually and forgot. I initially added an await and it caused a huge chain of things needing to be changed, so I added this principally to make a warning go away in my IDE as we weren't waiting. This is one where we don't really need to wait, so can just take this back out